src/subcommand.rs

```diff
@@ -493,19 +485,12 @@ impl Subcommand {
         let mut line_width = UnicodeWidthStr::width(*name);
 
         for parameter in &recipe.parameters {
-          line_width += if let Some(evaluator) = &mut evaluator {
-            UnicodeWidthStr::width(
-              format!(
-                " {}",
-                parameter
-                  .format_eval(evaluator)
-                  .color_display(Color::never())
-              )
-              .as_str(),
-            )
-          } else {
-            UnicodeWidthStr::width(format!(" {}", parameter.color_display(Color::never())).as_str())
+          if let Some(evaluator) = &mut evaluator {
+            let parameter = parameter.evaluate_default(evaluator)?;
```

Comment:
According to the borrow checker, we're borrowing from `&justfile.recipes` in line 479 when we return in the `Err` case here. Despite the only lifetime being mentioned in the return value of `evalaute_default` being `'src`. I'm not sure why this is, as it works fine in `justfile.run()`.

Reply:
Changing the `justfile` parameter of `list` to not be moved but borrowed, yields a similar result, requiring the borrow of `&Justfile<'scr>` itself to outlive `'src`, i.e., suggesting `&'src Justfile<'src>`, which is not at all satiable.